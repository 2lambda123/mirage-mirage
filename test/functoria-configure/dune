(alias
  (name runtest)
  (package functoria)
  (deps (package functoria) (source_tree src))
  (action (progn
    (run %{exe:../f0/f0_run.exe} configure -f src/config.ml)
    (diff? src/main.ml.expected src/main.ml)
    (diff? src/dune.expected src/dune)
    (diff? src/dune.build.expected src/dune.build)
    (diff? src/dune.config.expected src/dune.config)
    (diff? src/.mirage.config.expected src/.mirage.config)
    (diff? src/key_gen.ml.expected src/key_gen.ml))))
