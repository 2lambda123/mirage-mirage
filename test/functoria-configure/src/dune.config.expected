;; Generated by "test config".

(executable
  (name config)
  (modules config)
  (libraries functoria.f0))
